aiosparql/client.py

```diff
@@ -166,3 +169,20 @@ def _pretty_print_query(self, query: str) -> str:
                     code=exc.code, message=exc.message, headers=exc.headers,
                     explanation=explanation)
             return await resp.json()
+
+    @property
+    def closed(self):
+        return self._closed
+
+    @asyncio.coroutine
+    def close(self):
+        self._closed = True
+        # NOTE: TypeError: object _CoroGuard can't be used in 'await'
```

Comment:
Yes I definitely want to but I need to isolate the problem from aiosparql first. At first glance I kinda fear that we will have to convert the session's `close()` of aiohttp to normal coroutine to fix the issue but that will break compatibility.